types/src/account_address.rs

```diff
@@ -164,6 +164,7 @@ pub fn create_stake_pool_address(
     full_seed.extend(bcs::to_bytes(&operator).unwrap());
     full_seed.extend(SALT);
     full_seed.extend(seed);
+    full_seed.push(255);
```

Comment:
This is redundant

Reply:
Why is this redundant? Without this, the tests fail.